modulefiles/build.wcoss_cray.intel

```diff
@@ -31,9 +31,9 @@ setenv ZLIB_ROOT /usrx/local/prod/zlib/1.2.7/intel/haswell
 setenv PNG_ROOT /usrx/local/prod/png/1.2.49/intel/haswell
 setenv Jasper_ROOT /usrx/local/prod/jasper/1.900.1/intel/haswell
 
-module use /gpfs/hps3/emc/nems/noscrub/emc.nemspara/soft/modulefiles
-#module load esmf/8.0.0
-setenv ESMFMKFILE /gpfs/hps3/emc/nems/noscrub/emc.nemspara/soft/esmf/8.0.0/lib/esmf.mk
+#module use /gpfs/hps3/emc/nems/noscrub/emc.nemspara/soft/modulefiles
```

Comment:
Instead of commenting out these lines, you can just remove them.

Reply:
These lines have been removed.